help/include/automatically-follow-topics.md

```diff
@@ -0,0 +1,10 @@
+{start_tabs}
+
+{settings_tab|notifications}
+
+{tab|desktop-web}
+
```

Comment:
In the screenshot, this seems to be missing the instruction for going to settings. Are the  settings_tab and tab in the wrong order?

Reply:
Same applied for automatically unmuting